.pre-commit-config.yaml

```
@@ -0,0 +1,14 @@
+default_language_version:
+  python: python3.8
+repos:
+  - repo: local
+    hooks:
+      - id: make-lint-diff
+        name: make lint-diff
+        entry: /usr/bin/make lint-diff
```

Unfortunately this won't work for the docker folks; let's dockerize it:

```suggestion
        entry: docker-compose run --rm web make lint-diff
```